src/app/controllers/conference_controller.py

```diff
@@ -148,4 +98,3 @@ def stopVideoProcessor(self):
         if self.__videoProcessor and self.__videoProcessor.isRunning:
```

Comment:
send error message saying the video processor is already running if it is running in the other page